services/director-v2/tests/integration/test_computation_api.py

```diff
@@ -142,7 +142,11 @@ def test_empty_computation(
     empty_project = project()
     response = client.post(
         COMPUTATION_URL,
-        json={"user_id": user_id, "project_id": str(empty_project.uuid)},
+        json={
```

Comment:
this is the same discussion as the other day with operations like deleting an object that does not exist. Why running an empty study would return failure? should it just run and produce no results and have running time zero instead?

Reply:
it returns a 422. which means unprocessable entity. you cannot start a computational task without computational tasks. that's all. there are a lot of examples like this.

Reply:
in your APIs, if you pass an empty parameter, you get a 422... it's the same